pilot/copytool/rucio.py

```diff
@@ -44,6 +46,7 @@ def copy_in(files, **kwargs):
         :raise: PilotException in case of controlled error
     """
 
+    trace_object = TraceReport()
```

Comment:
Introducing new `TraceReport` instance  `trace_object`  is redundant and actually useless, why do you need it?

there is already properly filled instance of `TraceReport` available via passed  `trace_report`  variable to a copytool module by top workflow

Reply:
trace_report was a string, at least in previous versions, not the TraceReport() object.

Reply:
sorry, trace_report is dict, not string. However, it is not TraceReport object.

Reply:
sorry, but you in the tests initialized `trace_report` as a dictionary:)

https://github.com/PanDAWMS/pilot2/compare/91fd4976d283e0c50c67c88f40b5fdbbde060cb9..4e8567e17b2ff4efc484d771e59ace73a2de3f98#diff-f722e8348b3c3f6c0af1560646e475e2L124

Reply:
sorry, stupid mistake